Reverb egs

sw005320 opened the pull request:
I cleaned up the REVERB challenge recipe a little bit by
- Put the data location variables from corpus.sh to run.sh to follow the other recipe conventions. Now we don't need corpus.sh
- Removed some experiments that are not important
- Changed the RESULTS and result summarization script accordingly.
- Modified --cmd appropriately.

danpovey commented:
Thanks!  Merging.

naxingyu commented:
Thanks Shinji. Does this recipe reproduce your submission to the REVERB 
challenge?

在 2016/2/3 9:34, Shinji Watanabe 写道:
>
> I cleaned up the REVERB challenge recipe a little bit by
>
>   * Put the data location variables from corpus.sh to run.sh to follow
>     the other recipe conventions. Now we don't need corpus.sh
>   * Removed some experiments that are not important
>   * Changed the RESULTS and result summarization script accordingly.
>   * Modified --cmd appropriately.
>
> ------------------------------------------------------------------------
>
>
>         You can view, comment on, or merge this pull request online at:
>
> https://github.com/kaldi-asr/kaldi/pull/481
>
>
>         Commit Summary
>
>   * Modified REVERB run.sh to follow the convention of the other recipe
>   * Added MERL database environment.
>   * Merge remote-tracking branch 'origin/master' into reverb-egs
>   * cleaning up run.sh
>   * bug fix
>   * fixed duplication bugs for fmllr basis computation
>   * Modified result summarization script
>   * Add --stage to get results
>   * Merge branch 'master' into reverb-egs
>   * changed mode and fixed bugs.
>
>
>         File Changes
>
>   * *M* egs/reverb/s5/RESULTS
>     <https://github.com/kaldi-asr/kaldi/pull/481/files#diff-0> (450)
>   * *D* egs/reverb/s5/corpus.sh
>     <https://github.com/kaldi-asr/kaldi/pull/481/files#diff-1> (17)
>   * *T* egs/reverb/s5/local/Generate_mcTrainData_cut.m
>     <https://github.com/kaldi-asr/kaldi/pull/481/files#diff-2> (0)
>   * *M* egs/reverb/s5/local/REVERB_mcwsjav_data_prep.sh
>     <https://github.com/kaldi-asr/kaldi/pull/481/files#diff-3> (20)
>   * *M* egs/reverb/s5/local/REVERB_wsjcam0_data_prep.sh
>     <https://github.com/kaldi-asr/kaldi/pull/481/files#diff-4> (10)
>   * *A* egs/reverb/s5/local/calc_wer.sh
>     <https://github.com/kaldi-asr/kaldi/pull/481/files#diff-5> (55)
>   * *M* egs/reverb/s5/local/get_results.sh
>     <https://github.com/kaldi-asr/kaldi/pull/481/files#diff-6> (24)
>   * *D* egs/reverb/s5/local/summarize_results.pl
>     <https://github.com/kaldi-asr/kaldi/pull/481/files#diff-7> (122)
>   * *M* egs/reverb/s5/run.sh
>     <https://github.com/kaldi-asr/kaldi/pull/481/files#diff-8> (463)
>   * *M* egs/wsj/s5/cmd.sh
>     <https://github.com/kaldi-asr/kaldi/pull/481/files#diff-9> (3)
>
>
>         Patch Links:
>
>   * https://github.com/kaldi-asr/kaldi/pull/481.patch
>   * https://github.com/kaldi-asr/kaldi/pull/481.diff
>
> —
> Reply to this email directly or view it on GitHub 
> <https://github.com/kaldi-asr/kaldi/pull/481>.
>

sw005320 commented:
Yes, I confirmed it in the MERL and JHU clusters.
But our REVERB challenge system also includes the BLSTM based
de-reverberation and system combination, which cannot be included in the
recipe.

On Tue, Feb 2, 2016 at 9:05 PM, Xingyu Na <notifications@github.com> wrote:

> Thanks Shinji. Does this recipe reproduce your submission to the REVERB
> challenge?
>
> 在 2016/2/3 9:34, Shinji Watanabe 写道:
> >
> > I cleaned up the REVERB challenge recipe a little bit by
> >
> > * Put the data location variables from corpus.sh to run.sh to follow
> > the other recipe conventions. Now we don't need corpus.sh
> > * Removed some experiments that are not important
> > * Changed the RESULTS and result summarization script accordingly.
> > * Modified --cmd appropriately.
> >
> > ------------------------------------------------------------------------
> >
> >
> > You can view, comment on, or merge this pull request online at:
> >
> > https://github.com/kaldi-asr/kaldi/pull/481
> >
> >
> > Commit Summary
> >
> > * Modified REVERB run.sh to follow the convention of the other recipe
> > * Added MERL database environment.
> > * Merge remote-tracking branch 'origin/master' into reverb-egs
> > * cleaning up run.sh
> > * bug fix
> > * fixed duplication bugs for fmllr basis computation
> > * Modified result summarization script
> > * Add --stage to get results
> > * Merge branch 'master' into reverb-egs
> > * changed mode and fixed bugs.
> >
> >
> > File Changes
> >
> > * *M* egs/reverb/s5/RESULTS
> > <https://github.com/kaldi-asr/kaldi/pull/481/files#diff-0> (450)
> > * *D* egs/reverb/s5/corpus.sh
> > <https://github.com/kaldi-asr/kaldi/pull/481/files#diff-1> (17)
> > * *T* egs/reverb/s5/local/Generate_mcTrainData_cut.m
> > <https://github.com/kaldi-asr/kaldi/pull/481/files#diff-2> (0)
> > * *M* egs/reverb/s5/local/REVERB_mcwsjav_data_prep.sh
> > <https://github.com/kaldi-asr/kaldi/pull/481/files#diff-3> (20)
> > * *M* egs/reverb/s5/local/REVERB_wsjcam0_data_prep.sh
> > <https://github.com/kaldi-asr/kaldi/pull/481/files#diff-4> (10)
> > * *A* egs/reverb/s5/local/calc_wer.sh
> > <https://github.com/kaldi-asr/kaldi/pull/481/files#diff-5> (55)
> > * *M* egs/reverb/s5/local/get_results.sh
> > <https://github.com/kaldi-asr/kaldi/pull/481/files#diff-6> (24)
> > * *D* egs/reverb/s5/local/summarize_results.pl
> > <https://github.com/kaldi-asr/kaldi/pull/481/files#diff-7> (122)
> > * *M* egs/reverb/s5/run.sh
> > <https://github.com/kaldi-asr/kaldi/pull/481/files#diff-8> (463)
> > * *M* egs/wsj/s5/cmd.sh
> > <https://github.com/kaldi-asr/kaldi/pull/481/files#diff-9> (3)
> >
> >
> > Patch Links:
> >
> > * https://github.com/kaldi-asr/kaldi/pull/481.patch
> > * https://github.com/kaldi-asr/kaldi/pull/481.diff
> >
> > —
> > Reply to this email directly or view it on GitHub
> > <https://github.com/kaldi-asr/kaldi/pull/481>.
>
> >
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/kaldi-asr/kaldi/pull/481#issuecomment-178957119>.
>



-- 
Shinji Watanabe
